### ISSUE TITLE
torch.compile(model) error during training: no file cuda.h

### ISSUE THREAD

**Issue by beegica** [pytorch/pytorch#90178]
### 🐛 Describe the bug

I tried compiling a model that works fine on the GPU without it. Linux system. Compiling the model seems to have no issue, but once you try to train the model it breaks. I can't quite figure out what to do here, another user suggested adding a cuda.h to the CPATH but that wasn't the solution. 

Thanks!

FULL TRACE: 
```
/tmp/tmpypa57lhg/main.c:2:10: fatal error: cuda.h: No such file or directory
    2 | #include "cuda.h"
      |          ^~~~~~~~
compilation terminated.

---------------------------------------------------------------------------
_RemoteTraceback                          Traceback (most recent call last)
_RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/bgj5hk/anaconda3/lib/python3.9/concurrent/futures/process.py", line 243, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
  File "/home/bgj5hk/anaconda3/lib/python3.9/site-packages/torch/_inductor/codecache.py", line 488, in _worker_compile
    kernel.precompile(warm_cache_only_with_cc=cc)
  File "/home/bgj5hk/anaconda3/lib/python3.9/site-packages/torch/_inductor/triton_ops/autotune.py", line 59, in precompile
    self.launchers = [
  File "/home/bgj5hk/anaconda3/lib/python3.9/site-packages/torch/_inductor/triton_ops/autotune.py", line 60, in <listcomp>
    self._precompile_config(c, warm_cache_only_with_cc)
  File "/home/bgj5hk/anaconda3/lib/python3.9/site-packages/torch/_inductor/triton_ops/autotune.py", line 74, in _precompile_config
    triton.compile(
  File "/home/bgj5hk/anaconda3/lib/python3.9/site-packages/triton/compiler.py", line 1239, in compile
    so = _build(fn.__name__, src_path, tmpdir)
  File "/home/bgj5hk/anaconda3/lib/python3.9/site-packages/triton/compiler.py", line 1169, in _build
    ret = subprocess.check_call(cc_cmd)
  File "/home/bgj5hk/anaconda3/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/usr/bin/gcc', '/tmp/tmpypa57lhg/main.c', '-O3', '-I/usr/local/cuda/include', '-I/home/bgj5hk/anaconda3/include/python3.9', '-I/tmp/tmpypa57lhg', '-shared', '-fPIC', '-lcuda', '-o', '/tmp/tmpypa57lhg/triton_.cpython-39-x86_64-linux-gnu.so', '-L/usr/lib/x86_64-linux-gnu']' returned non-zero exit status 1.
"""

The above exception was the direct cause of the following exception:

CalledProcessError                        Traceback (most recent call last)
File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/output_graph.py:546, in OutputGraph.call_user_compiler(self, gm)
    545     compiler_fn = wrap_compiler_fn(compiler_fn)
--> 546 compiled_fn = compiler_fn(gm, self.example_inputs())
    547 _step_logger()(logging.INFO, f"done compiler function {name}")

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/debug_utils.py:911, in wrap_backend_debug.<locals>.debug_wrapper(gm, example_inputs, **kwargs)
    910 else:
--> 911     compiled_gm = compiler_fn(gm, example_inputs, **kwargs)
    913 return compiled_gm

File ~/anaconda3/lib/python3.9/site-packages/torch/__init__.py:1177, in compile.<locals>._compile_fn(model_, inputs_)
   1176 with cm:
-> 1177     return compile_fn(model_, inputs_)

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/compile_fx.py:398, in compile_fx(model_, example_inputs_, inner_compile)
    393 with overrides.patch_functions():
    394 
    395     # TODO: can add logging before/after the call to create_aot_dispatcher_function
    396     # in torch._functorch/aot_autograd.py::aot_module_simplified::aot_function_simplified::new_func
    397     # once torchdynamo is merged into pytorch
--> 398     return aot_autograd(
    399         fw_compiler=fw_compiler,
    400         bw_compiler=bw_compiler,
    401         decompositions=select_decomp_table(),
    402         partition_fn=functools.partial(
    403             min_cut_rematerialization_partition, compiler="inductor"
    404         ),
    405         # A "tiny" graph can actually decompose into multiple
    406         # operators (if it's a decomposition) and inductor can
    407         # do a better job on it in this case
    408         #
    409         # Also, for some reason, test_comprehensive___rmatmul___cpu
    410         # fails without forcing a compile lol.
    411         force_compile_tiny_graphs=True,
    412     )(model_, example_inputs_)

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/optimizations/training.py:80, in aot_autograd.<locals>.compiler_fn(gm, example_inputs)
     78 try:
     79     # NB: NOT cloned!
---> 80     cg = aot_module_simplified(gm, example_inputs, **kwargs)
     81     counters["aot_autograd"]["ok"] += 1

File ~/anaconda3/lib/python3.9/site-packages/torch/_functorch/aot_autograd.py:2093, in aot_module_simplified(mod, args, fw_compiler, bw_compiler, partition_fn, decompositions, hasher_type, static_argnums)
   2091 full_args.extend(args)
-> 2093 compiled_fn = create_aot_dispatcher_function(
   2094     functional_call,
   2095     full_args,
   2096     aot_config,
   2097 )
   2099 # TODO: There is something deeply wrong here; compiled_fn running with
   2100 # the boxed calling convention, but aot_module_simplified somehow
   2101 # historically returned a function that was not the boxed calling
   2102 # convention.  This should get fixed...

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/utils.py:90, in dynamo_timed.<locals>.time_wrapper(*args, **kwargs)
     89 t0 = time.time()
---> 90 r = func(*args, **kwargs)
     91 latency = time.time() - t0

File ~/anaconda3/lib/python3.9/site-packages/torch/_functorch/aot_autograd.py:1792, in create_aot_dispatcher_function(flat_fn, flat_args, aot_config)
   1790 # You can put more passes here
-> 1792 compiled_fn = compiler_fn(flat_fn, fake_flat_tensor_args, aot_config)
   1794 if not hasattr(compiled_fn, '_boxed_call'):

File ~/anaconda3/lib/python3.9/site-packages/torch/_functorch/aot_autograd.py:1197, in aot_wrapper_dedupe(flat_fn, flat_args, aot_config, compiler_fn)
   1196     if ok:
-> 1197         return compiler_fn(flat_fn, leaf_flat_args, aot_config)
   1199 # Strategy 2: Duplicate specialize.
   1200 #
   1201 # In Haskell types, suppose you have:
   (...)
   1233 #   }
   1234 #   keep_arg_mask = [True, True, False, True]

File ~/anaconda3/lib/python3.9/site-packages/torch/_functorch/aot_autograd.py:877, in aot_dispatch_base(flat_fn, flat_args, aot_config)
    876 with context(), track_graph_compiling(aot_config, "inference"):
--> 877     compiled_fw = aot_config.fw_compiler(fw_module, flat_args)
    879 @wraps(compiled_fw)
    880 def new_fn(args):

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/utils.py:90, in dynamo_timed.<locals>.time_wrapper(*args, **kwargs)
     89 t0 = time.time()
---> 90 r = func(*args, **kwargs)
     91 latency = time.time() - t0

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/compile_fx.py:373, in compile_fx.<locals>.fw_compiler(model, example_inputs)
    372 fixed = len(example_inputs) - num_example_inputs
--> 373 return inner_compile(
    374     model,
    375     example_inputs,
    376     num_fixed=fixed,
    377     cudagraphs=cudagraphs,
    378     graph_id=graph_id,
    379 )

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/debug_utils.py:494, in wrap_compiler_debug.<locals>.debug_wrapper(gm, example_inputs, **kwargs)
    493 else:
--> 494     compiled_fn = compiler_fn(gm, example_inputs, **kwargs)
    496 return compiled_fn

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/debug.py:177, in DebugContext.wrap.<locals>.inner(*args, **kwargs)
    176 with DebugContext():
--> 177     return fn(*args, **kwargs)

File ~/anaconda3/lib/python3.9/contextlib.py:79, in ContextDecorator.__call__.<locals>.inner(*args, **kwds)
     78 with self._recreate_cm():
---> 79     return func(*args, **kwds)

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/compile_fx.py:140, in compile_fx_inner(gm, example_inputs, cudagraphs, num_fixed, is_backward, graph_id)
    139     graph.run(*example_inputs)
--> 140     compiled_fn = graph.compile_to_fn()
    142 if cudagraphs:

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/graph.py:500, in GraphLowering.compile_to_fn(self)
    499 def compile_to_fn(self):
--> 500     return self.compile_to_module().call

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/utils.py:90, in dynamo_timed.<locals>.time_wrapper(*args, **kwargs)
     89 t0 = time.time()
---> 90 r = func(*args, **kwargs)
     91 latency = time.time() - t0

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/graph.py:490, in GraphLowering.compile_to_module(self)
    488     print(code)
--> 490 mod = PyCodeCache.load(code)
    491 for name, value in self.constants.items():

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/codecache.py:459, in PyCodeCache.load(cls, source_code)
    458 mod.key = key
--> 459 exec(code, mod.__dict__, mod.__dict__)
    460 # another thread might set this first

File /tmp/torchinductor_bgj5hk/25/c257l6iwet7jbrnvyakofp7qdondxx4cpnx4o5j4p6rerp7akjhn.py:39, in <module>
     18 triton_fused__to_copy_zeros_0 = async_compile.triton('''
     19 import triton
     20 import triton.language as tl
   (...)
     35     tl.store(out_ptr0 + (x0 + tl.zeros([XBLOCK], tl.int32)), tmp0, xmask)
     36 ''')
---> 39 async_compile.wait(globals())
     40 del async_compile

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/codecache.py:642, in AsyncCompile.wait(self, scope)
    641         if isinstance(result, (Future, TritonFuture)):
--> 642             scope[key] = result.result()
    644 _compile_end()

File ~/anaconda3/lib/python3.9/site-packages/torch/_inductor/codecache.py:512, in TritonFuture.result(self)
    511 # If the worker failed this will throw an exception.
--> 512 self.future.result()
    513 kernel = self.kernel = _load_kernel(self.source_code)

File ~/anaconda3/lib/python3.9/concurrent/futures/_base.py:446, in Future.result(self, timeout)
    445 elif self._state == FINISHED:
--> 446     return self.__get_result()
    447 else:

File ~/anaconda3/lib/python3.9/concurrent/futures/_base.py:391, in Future.__get_result(self)
    390 try:
--> 391     raise self._exception
    392 finally:
    393     # Break a reference cycle with the exception in self._exception

CalledProcessError: Command '['/usr/bin/gcc', '/tmp/tmpypa57lhg/main.c', '-O3', '-I/usr/local/cuda/include', '-I/home/bgj5hk/anaconda3/include/python3.9', '-I/tmp/tmpypa57lhg', '-shared', '-fPIC', '-lcuda', '-o', '/tmp/tmpypa57lhg/triton_.cpython-39-x86_64-linux-gnu.so', '-L/usr/lib/x86_64-linux-gnu']' returned non-zero exit status 1.

The above exception was the direct cause of the following exception:

BackendCompilerFailed                     Traceback (most recent call last)
Input In [11], in <cell line: 1>()
      1 for epoch in range(curr_epoch, epochs):
      2     epoch_start_time = time.time()
----> 3     train_loss, train_acc = train(model=model, loader=train_loader, 
      4                                   optimizer=optimizer,criterion=criterion, 
      5                                   batch_size=batch_size, 
      6                                   log_interval=prog_upd_int, 
      7                                   fixations=fixations,
      8                                   best_val_acc=best_val_acc, 
      9                                   val_acc=val_acc, 
     10                                   val_loss=val_loss,
     11                                   perf_file=trainlog_fn, epoch=epoch,
     12                                   scheduler=lr_scheduler,
     13                                   device=device, wandb=wandb)
     14     val_loss, val_acc = test(model=model, loader=test_loader, 
     15                              criterion=criterion, fixations=fixations,
     16                              perf_file=validlog_fn,
     17                              wandb=wandb, device=device)
     18     epoch_time = (time.time() - epoch_start_time)

File ~/github/SITHEyeCon/MNIST/training.py:44, in train(model, loader, optimizer, criterion, batch_size, log_interval, fixations, best_val_acc, val_acc, val_loss, perf_file, epoch, scheduler, device, wandb)
     42 fix_loc = np.array(fix_convert(fixations, data.shape[-2:]))
     43 fix_loc = fix_loc[:, None].repeat(data.shape[0], 1)
---> 44 output = model(data, fix_loc)
     46 # Calculate loss
     47 curr_loss = criterion(output, target)

File ~/anaconda3/lib/python3.9/site-packages/torch/nn/modules/module.py:1480, in Module._call_impl(self, *args, **kwargs)
   1475 # If we don't have any hooks, we want to skip the rest of the logic in
   1476 # this function, and just call forward.
   1477 if not (self._backward_hooks or self._backward_pre_hooks or self._forward_hooks or self._forward_pre_hooks
   1478         or _global_backward_pre_hooks or _global_backward_hooks
   1479         or _global_forward_hooks or _global_forward_pre_hooks):
-> 1480     return forward_call(*args, **kwargs)
   1481 # Do not call functions when jit is used
   1482 full_backward_hooks, non_full_backward_hooks = [], []

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/eval_frame.py:80, in OptimizedModule.forward(self, *args, **kwargs)
     79 def forward(self, *args, **kwargs):
---> 80     return self.dynamo_ctx(self._orig_mod.forward)(*args, **kwargs)

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/eval_frame.py:209, in _TorchDynamoContext.__call__.<locals>._fn(*args, **kwargs)
    207 dynamic_ctx.__enter__()
    208 try:
--> 209     return fn(*args, **kwargs)
    210 finally:
    211     set_eval_frame(prior)

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/eval_frame.py:329, in catch_errors_wrapper.<locals>.catch_errors(frame, cache_size)
    326             return hijacked_callback(frame, cache_size)
    328 with compile_lock:
--> 329     return callback(frame, cache_size)

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py:468, in convert_frame.<locals>._convert_frame(frame, cache_size)
    466 counters["frames"]["total"] += 1
    467 try:
--> 468     result = inner_convert(frame, cache_size)
    469     counters["frames"]["ok"] += 1
    470     return result

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py:102, in wrap_convert_context.<locals>._fn(*args, **kwargs)
    100 torch.fx.graph_module._forward_from_src = fx_forward_from_src_skip_result
    101 try:
--> 102     return fn(*args, **kwargs)
    103 finally:
    104     torch._C._set_grad_enabled(prior_grad_mode)

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/utils.py:90, in dynamo_timed.<locals>.time_wrapper(*args, **kwargs)
     88     compilation_metrics[key] = []
     89 t0 = time.time()
---> 90 r = func(*args, **kwargs)
     91 latency = time.time() - t0
     92 # print(f"Dynamo timer: key={key}, latency={latency:.2f} sec")

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py:339, in convert_frame_assert.<locals>._convert_frame_assert(frame, cache_size)
    336 global initial_grad_state
    337 initial_grad_state = torch.is_grad_enabled()
--> 339 return _compile(
    340     frame.f_code,
    341     frame.f_globals,
    342     frame.f_locals,
    343     frame.f_builtins,
    344     compiler_fn,
    345     one_graph,
    346     export,
    347     guard_export_fn,
    348     frame,
    349 )

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py:395, in _compile(code, globals, locals, builtins, compiler_fn, one_graph, export, guard_export_fn, frame)
    393 for attempt in itertools.count():
    394     try:
--> 395         out_code = transform_code_object(code, transform)
    396         orig_code_map[out_code] = code
    397         break

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/bytecode_transformation.py:341, in transform_code_object(code, transformations, safe)
    338 instructions = cleaned_instructions(code, safe)
    339 propagate_line_nums(instructions)
--> 341 transformations(instructions, code_options)
    343 fix_vars(instructions, code_options)
    345 dirty = True

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py:382, in _compile.<locals>.transform(instructions, code_options)
    370 nonlocal output
    371 tracer = InstructionTranslator(
    372     instructions,
    373     code,
   (...)
    380     export,
    381 )
--> 382 tracer.run()
    383 output = tracer.output
    384 assert output is not None

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/symbolic_convert.py:1625, in InstructionTranslator.run(self)
   1623 def run(self):
   1624     _step_logger()(logging.INFO, f"torchdynamo start tracing {self.f_code.co_name}")
-> 1625     super().run()

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/symbolic_convert.py:484, in InstructionTranslatorBase.run(self)
    479 def run(self):
    480     try:
    481         while (
    482             self.instruction_pointer is not None
    483             and not self.output.should_exit
--> 484             and self.step()
    485         ):
    486             pass
    487     except BackendCompilerFailed:

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/symbolic_convert.py:454, in InstructionTranslatorBase.step(self)
    452     if not hasattr(self, inst.opname):
    453         unimplemented(f"missing: {inst.opname}")
--> 454     getattr(self, inst.opname)(inst)
    456     return inst.opname != "RETURN_VALUE"
    457 except BackendCompilerFailed:

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/symbolic_convert.py:308, in break_graph_if_unsupported.<locals>.decorator.<locals>.wrapper(self, inst)
    306     reason = GraphCompileReason(excp.msg, user_stack)
    307 self.restore_graphstate(state)
--> 308 self.output.compile_subgraph(self, reason=reason)
    309 self.popn(push - dis.stack_effect(inst.opcode, inst.arg))
    311 for _ in range(push):

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/output_graph.py:464, in OutputGraph.compile_subgraph(self, tx, partial_convert, reason)
    461 output = []
    462 if count_calls(self.graph) != 0 or len(pass2.graph_outputs) != 0:
    463     output.extend(
--> 464         self.compile_and_call_fx_graph(tx, pass2.graph_output_vars(), root)
    465     )
    467     if len(pass2.graph_outputs) != 0:
    468         output.append(pass2.create_store(graph_output_var))

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/output_graph.py:511, in OutputGraph.compile_and_call_fx_graph(self, tx, rv, root)
    508 name = unique_id("__compiled_fn")
    510 assert_no_fake_params_or_buffers(gm)
--> 511 compiled_fn = self.call_user_compiler(gm)
    512 compiled_fn = disable(compiled_fn)
    514 counters["stats"]["unique_graphs"] += 1

File ~/anaconda3/lib/python3.9/site-packages/torch/_dynamo/output_graph.py:551, in OutputGraph.call_user_compiler(self, gm)
    549 except Exception as e:
    550     compiled_fn = gm.forward
--> 551     raise BackendCompilerFailed(self.compiler_fn, e) from e
    552 return compiled_fn

BackendCompilerFailed: _compile_fn raised CalledProcessError: Command '['/usr/bin/gcc', '/tmp/tmpypa57lhg/main.c', '-O3', '-I/usr/local/cuda/include', '-I/home/bgj5hk/anaconda3/include/python3.9', '-I/tmp/tmpypa57lhg', '-shared', '-fPIC', '-lcuda', '-o', '/tmp/tmpypa57lhg/triton_.cpython-39-x86_64-linux-gnu.so', '-L/usr/lib/x86_64-linux-gnu']' returned non-zero exit status 1.

Set torch._dynamo.config.verbose=True for more information


You can suppress this exception and fall back to eager by setting:
    torch._dynamo.config.suppress_errors = True
```

### Versions

Collecting environment information...
PyTorch version: 1.14.0.dev20221205+cu117
Is debug build: False
CUDA used to build PyTorch: 11.7
ROCM used to build PyTorch: N/A

OS: Debian GNU/Linux bookworm/sid (x86_64)
GCC version: (Debian 12.2.0-9) 12.2.0
Clang version: Could not collect
CMake version: version 3.25.0
Libc version: glibc-2.36

Python version: 3.9.12 (main, Apr  5 2022, 06:56:58)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.19.0-2-amd64-x86_64-with-glibc2.36
Is CUDA available: True
CUDA runtime version: 11.6.124
CUDA_MODULE_LOADING set to: LAZY
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 3090
Nvidia driver version: 510.85.02
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.24.0rc2
[pip3] numpydoc==1.2
[pip3] torch==1.14.0.dev20221205+cu117
[pip3] torch-scatter==2.0.9
[pip3] torch-sparse==0.6.15
[pip3] torchaudio==0.14.0.dev20221205+cu117
[pip3] torchdata==0.4.1
[pip3] torchtext==0.13.1
[pip3] torchtriton==2.0.0+0d7e753227
[pip3] torchvision==0.15.0.dev20221204+cpu
[pip3] torchviz==0.0.2
[conda] blas                      1.0                         mkl  
[conda] cudatoolkit               11.3.1               h2bc3f7f_2  
[conda] mkl                       2021.4.0           h06a4308_640  
[conda] mkl-service               2.4.0            py39h7f8727e_0  
[conda] mkl_fft                   1.3.1            py39hd3c417c_0  
[conda] mkl_random                1.2.2            py39h51133e4_0  
[conda] numpy                     1.24.0rc2                pypi_0    pypi
[conda] numpydoc                  1.2                pyhd3eb1b0_0  
[conda] pytorch-cuda              11.6                 h867d48c_0    pytorch-nightly
[conda] pytorch-mutex             1.0                        cuda    pytorch
[conda] pytorch-scatter           2.0.9           py39_torch_1.12.0_cu113    pyg
[conda] pytorch-sparse            0.6.15          py39_torch_1.12.0_cu113    pyg
[conda] torch                     1.14.0.dev20221205+cu117          pypi_0    pypi
[conda] torchaudio                0.14.0.dev20221205+cu117          pypi_0    pypi
[conda] torchdata                 0.4.1                    pypi_0    pypi
[conda] torchtext                 0.13.1                   pypi_0    pypi
[conda] torchtriton               2.0.0+0d7e753227          pypi_0    pypi
[conda] torchvision               0.15.0.dev20221204+cpu          pypi_0    pypi
[conda] torchviz                  0.0.2                    pypi_0    pypi

cc @ezyang @soumith

**Comment by beegica** [comment]
Okay hmm. Yeah despite 3 different cuda sources, none of them create a cuda folder. Let me get rid of all of those and reinstall cuda? I think I can try that.

**Comment by msaroufim** [comment]
The nightly docker images are a good option if you don't want to deal with CUDA installation

```
docker pull ghcr.io/pytorch/pytorch-nightly
# And for ad hoc experiments just make sure that your container has access to all your GPUs:
docker run --gpus all -it ghcr.io/pytorch/pytorch-nightly:latest /bin/bash
```

**Comment by wconstab** [comment]
@beegica were you able to get a working cuda install?  If you could close out this issue or post follow-ups questions that would help us out.

**Comment by beegica** [comment]
Cuda now works! but I have a different issue Now. close this up and ill post another issue! 

Thanks for your help!
